src/main/java/training/Wrist/Wrist.java

```diff
@@ -0,0 +1,62 @@
+package training.Wrist;
+
+import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.can.BaseTalonConfiguration;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.can.TalonSRXConfiguration;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import utils.GBSubsystem;
+
+import static training.Wrist.WristConstants.*;
+
+public class Wrist extends GBSubsystem {
+    private static Wrist instance;
+
+    public static Wrist getInstance() {
+        if (instance == null) {
+            instance = new Wrist();
+        }
+        return instance;
+    }
+
+    private final TalonSRX motor;
+    private final TalonSRXConfiguration motorConfiguration;
+    private BaseTalonConfiguration configuration;
+    private Rotation2d targetAngle;
+
+    private Wrist() {
+        this.motor = new TalonSRX(WRIST_ID);
+        this.motorConfiguration = new TalonSRXConfiguration();
+        motor.configAllSettings(motorConfiguration);
+    }
+
+    @Deprecated // using rotate() and changing manually the constants in the calibration process is advised
+    public void setPower(double power) {
+        motor.set(TalonSRXControlMode.PercentOutput, power);
+        if (Math.abs(power) >= POWER_LIMIT_WRIST) {
+            SmartDashboard.putString("motor is trying to spin in power above MAX_POWER_CIM limit", "");
+        }
```

Comment:
maybe check before you set the power, so it wont try

Reply:
I think I'll remove this method since PID anyway overwrites it.

Reply:
Don’t delete it, you want set power function or set volt for manual control and low level testers